from gold_prediction.exception.exception import CustomException 